gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -268,21 +281,15 @@ run_initial_contact_quest (GisSummaryPage *page)
 }
 
 static void
-initial_contact_connect_to_clippy (GisSummaryPage *page)
+clippy_connect_called_cb (GObject      *proxy,
+                          GAsyncResult *res,
+                          gpointer      user_data)
 {
+  GisSummaryPage *page = GIS_SUMMARY_PAGE (user_data);
   GisSummaryPagePrivate *priv = gis_summary_page_get_instance_private (page);
   g_autoptr(GError) error = NULL;
 
-  g_dbus_proxy_call_sync (priv->clippy_proxy,
-                          "Connect",
-                          g_variant_new ("(sss)",
-                                         "view.JSContext.globalParameters",
-                                         "notify",
-                                         "unlocked"),
-                          G_DBUS_CALL_FLAGS_NONE,
-                          G_MAXINT,
-                          NULL,
-                          &error);
+  g_dbus_proxy_call_finish (G_DBUS_PROXY (proxy), res, &error);
```

Comment:
Don't we need to call `set_busy(FALSE)` at the end of this? Or does the cursor change back to normal as a consequence of this window getting hidden and the unlock app starting?

Reply:
Ah, I did it before but for some reason I removed it. But yeah, the cursor went back to normal without `set_busy (page, FALSE);`, probably because of the way I did it, the cursor was only set for the g-i-s window, not sure.

Updated it anyway, looks more correct to reset it.

Reply:
Cool! I don't mean to nitpick but we should probably also reset the cursor in the error code paths then.

Reply:
I added the `set_busy (page, FALSE);` line in `finish_fbe`, which is called on both the success and error path.

Reply:
I think, maybe I'm missing something.

Reply:
My bad!